dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -360,22 +359,84 @@ def test_merge_correlation(self, *mocks):
 
         expected_corr_mat = np.array([
             [1.0, -0.26559388521279237, 0.26594894270403086],
-            [-0.26559388521279237, 1.0, -0.4907232900448271],
-            [0.26594894270403086, -0.4907232900448271, 1.0]
+            [-0.26559388521279237, 1.0, -0.49072329],
+            [0.26594894270403086, -0.4907239, 1.0]
         ])
-        np.testing.assert_array_equal(expected_corr_mat,
+        np.testing.assert_array_almost_equal(expected_corr_mat,
                                       merged_profile.correlation_matrix)
 
-        # merge between an existing corr and None correlation (with data)
+    def test_correlation_update(self):
+        profile_options = dp.ProfilerOptions()
+        profile_options.set({"correlation.is_enabled": True})
```

Comment:
should we set correlation option to True as the update works now?
@JGSweets thoughts?

Reply:
Let's get a profiled time first before we enable by default.